### ISSUE TITLE
Final inspection

### ISSUE THREAD

**Issue by ghost** [RomanSaveljev/sonic-pi#51]
I am planning tomorrow (29.02.16) to finish the translation of the last article, so ripe the following question:
@RomanSaveljev, could you find the time to check the translation before merging with the main development branch? 
Besides, I recently noticed that not all the translated articles passed automatic verification

**Comment by RomanSaveljev** [comment]
Yes, I will try to find time to check the whole thing. Won't happen in one day, but I wil try to do it regularly. I think it is enough that the last commit passes. Besides, they are likely to squash all commits into one and merge it to the main line to keep the history clean.

**Comment by ghost** [comment]
OK, success!

**Comment by ghost** [comment]
Hi @RomanSaveljev! Inspection progressing? :-)

**Comment by RomanSaveljev** [comment]
Slowly...

**Comment by ghost** [comment]
Nothing to worry about. The main thing is not to stop. Every day to do some of the work. I'd checked the spelling, typos and stopped at this. Although the translation is far from perfection it can improving to infinity.

**Comment by RomanSaveljev** [comment]
Yes, I am doing it little by little every day. Thanks for the advice and sorry for it to be taking long.

**Comment by ghost** [comment]
No problems

**Comment by ghost** [comment]
I recently was faced with the fact, that the interface buttons is PNG files, and they are not translated. But in the tutorial I translated their names. What do you think, whether to modify the tutorial?
